Configuration/AlCa/python/autoCond.py

```diff
@@ -76,7 +76,7 @@
     # GlobalTag for MC production with realistic conditions for Phase1 2024
     'phase1_2024_realistic'        : '123X_mcRun3_2024_realistic_v6',
     # GlobalTag for MC production with realistic conditions for Phase2
-    'phase2_realistic'             : '123X_mcRun4_realistic_v4'
+    'phase2_realistic'             : '123X_mcRun4_realistic_noSiStrip_v3'
```

Comment:
I have a question about the versions in the names of the GT: do the versions of `123X_mcRun4_realistic` and `123X_mcRun4_realistic_noSiStrip` follow independent numerations? Or the intention was instead that `123X_mcRun4_realistic_noSiStrip_v3` should have meant `123X_mcRun4_realistic_v3` without SiStrips? In the second case, the version numbers should be synchronized. Otherwise, we can remain as it is now.

Reply:
Hi @perrotta when we do a next update of the autoCond we'll do a `123X_mcRun4_realistic_v5` that will be equivalent of the `123X_mcRun4_realistic_noSiStrip_v3`. I got a bit confused if this GT is for testing alone, that's why it has a specific name of `noSiStrip`. Let's keep it for this PR, we'll have anyway an autoCond update planned very soon

Reply:
@perrotta the content of the GT is correct (se PR description), the version number is irrelevant for the purpose of the PR. I could have put a candidate GT but that's what AlCa provided me.